Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift

```diff
@@ -181,7 +182,7 @@ class EdgeHitProcessor: HitProcessing {
         }
     }
 
-    /// Extracts all the Edge configuration keys from the Configuration shared state
+    /// Extracts all the Edge configuration keys needed for this request. Configuration keys are extracted from the `Configuration shared state` and the given `event` instance
```

Comment:
You don't need the back ticks around "Configuration shared state" here as it adds extra highlighting to the text. The reason why I suggested adding them to "event" is because it's the name of the parameter. For example, in Javadoc, it's common to refer to a parameter name with `{@code event}`, where Swift uses back ticks instead of the code annotation. 
Also add a period at the end of the sentence.